src/languages/vbnet.js

```diff
@@ -28,23 +28,41 @@ export default function(hljs) {
     ]
   };
 
-  /**
-   * Date Literals consist of a date, a time, or both separated by whitespace, surrounded by #.
-   */
+  /** Date Literals consist of a date, a time, or both separated by whitespace, surrounded by # */
   const DATE = {
     className: 'literal',
     variants: [
       {
-        // Date: #M/D/YYYY# or #YYYY-MM-DD#
-        begin: /# *((\d+[-/]){2}\d+) *#/
+        // #YYYY-MM-DD# (ISO-Date)
+        begin: /# *\d{4}(-\d\d){2} *#/
```

Comment:
Is `2000/1/1` not allowed?

Reply:
The docs seems inconclusive:
> You must specify the date value in the format M/d/yyyy, for example #5/31/1993#, or yyyy-MM-dd, for example #1993-5-31#.

The format would say "No", but the example says "Yes".

Changed it to be more inclusive though.